### PR TITLE
Adding initial usage docs

### PR DESCRIPTION
**Pull request by sekka1**

### CODE REVIEW

File: docs/observability.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Observability
+
+## Logs
+Loki
+
```
Comment (seethatgo) [inline docs/observability.md:5]: Some sample queries like `{namespace="myapp"}` for common troubleshooting / monitoring of container logs to provide an initial runbook of useful info an operator may need as they run the cluster / debug apps running on the cluster

File: docs/how-to-add-in-a-new-app.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# How to add in a new app
+
+
+If a new github repo is added, what all activities need to be done to deploy its code in production
```
Comment (seethatgo) [inline docs/how-to-add-in-a-new-app.md:4]: this is good - like a checklist of what's needed like folder for app, TF cloud workspace, github action to build container, github action to deploy container, permissions, etc.

**Comment by seethatgo** [comment]
Other example:
How to deploy sealed-secrets to cluster
How to expose sealed-secret as environment variable for app

**Comment by seethatgo** [comment]
How to add environment variables to an app using `helm_generic` chart

File: docs/cluster-operations.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## the IAM provisioning - what policies and roles are setup and how will it be maintained
+
+## How to scale up or scale down a POD
+
+## Start and restarting a POD
+
+## How to give a pod certain IAM policy permissions
+
+
```
Comment (sekka1) [inline docs/cluster-operations.md:13]: How to give a pod access to the RDS database via a pod IAM role?

**Pull request merged**